### PR TITLE
Implement readers.tindex

### PR DESCRIPTION
**Pull request by hobu**

### CODE REVIEW

File: include/pdal/SpatialReference.hpp [hunk 1]
```diff
@@ -75,10 +75,14 @@ class PDAL_DLL SpatialReference
     bool operator<(const SpatialReference& other) const
         { return m_wkt < other.m_wkt; }
 
-    // Returns true iff the object doesn't contain a valid srs.
+    // Returns true if the object doesn't contain a valid srs.
```
Comment (abellgithub) [inline include/pdal/SpatialReference.hpp]: The original is math-speak, not a typo, I don't think :)

File: io/las/LasReader.cpp [hunk 2]
```diff
@@ -67,6 +67,7 @@ class invalid_stream : public pdal_error
 
 void LasReader::processOptions(const Options& options)
 {
+    m_skipWktVLR = options.getValueOrDefault<bool>("skip_vlr_wkt", false);
```
Comment (abellgithub) [inline io/las/LasReader.cpp]: Don't think this is necessary, is it?  The code I'm going to commit for the other fix will take care of this.  Is this just a temp thing?

Comment (abellgithub) [inline io/las/LasReader.cpp]: Do tests pass with this?  Because mine fail, which is why the fix hasn't been merged.

Comment (hobu) [inline io/las/LasReader.cpp]: I think this is sloppy merging.

**Pull request closed without merging**